paasta_tools/chronos_tools.py

```diff
@@ -477,8 +473,8 @@ def create_complete_config(service, job_name, soa_dir=DEFAULT_SOA_DIR):
     # this makes it trivial to compare configs and know when to bounce.
     complete_config['description'] = get_config_hash(complete_config)
 
-    # If the job was previously stopped, we should stop the new job as well
-    # NOTE this clobbers the 'disabled' param specified in the config file!
+    # use the 'description' field to determine the disabled state
```

Comment:
This comment isn't correct anymore, I would just junk it. And then maybe move line 470 to line 477 to bring the related code closer together